ambry-replication/src/main/java/com.github.ambry.replication/ReplicationManager.java

```diff
@@ -622,7 +622,7 @@ private void readFromFileAndPersistIfNecessary(String mountPath) throws Replicat
                       && remoteReplicaInfo.getReplicaId().getReplicaPath().equals(replicaPath)) {
                     logger.info("Read token for partition {} remote host {} port {} token {}", partitionId, hostname,
                         port, token);
-                    if (partitionInfo.getStore().getSizeInBytes() > 0) {
+                    if (partitionInfo.getStore().isEmpty() == false) {
```

Comment:
minor: `if (!partitionInfo.getStore().isEmpty())`

Reply:
I realize that's the standard and I'll change it.  I do feel that the ! is easy to miss and an explicit false check is clearer.  That's why I prefer it.